package.json

```diff
@@ -46,35 +46,30 @@
     ]
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "emoji-regex": "^8.0.0",
+    "classnames": "^2.3.1",
+    "emoji-regex": "^10.1.0",
     "lodash.flatten": "^4.4.0",
-    "prop-types": "^15.5.8",
-    "string-replace-to-array": "^1.0.1"
+    "prop-types": "^15.8.1",
+    "string-replace-to-array": "^2.1.0"
   },
   "peerDependencies": {
-    "react": ">=0.14.0",
-    "react-dom": ">=0.14.0"
+    "react": ">=16",
+    "react-dom": ">=16"
```

Comment:
We want to keep the `0.14.0` versions requirements as there is no need for a more recent React version.